ocs_ci/ocs/scale_lib.py

```diff
@@ -284,7 +284,9 @@ def create_scale_pods(
         self.ms_name = list()
 
         # Check for expected worker count
-        if is_managed_service_cluster():
+        if is_hci_cluster():
+            expected_worker_count = 2
```

Comment:
Shouldn't these values be part of a deployment config?

Reply:
In the deployment config `conf/deployment/fusion_hci_pc/client_bm_upi_1az_rhcos_nvme_3w.yaml`, I saw there are 3 worker nodes, but in fact, there are only 2 worker nodes in the client cluster. Can we add this value to the constants?

Reply:
If this is the case then the config should be updated.

Reply:
Okay, we can talk about it with others.